### ISSUE TITLE
Doctrine SqlLogger does not log any queries

### ISSUE THREAD

**Issue by sebastiansommer** [neos/flow-development-collection#1854]
### Description
After updating to Flow 6.0 the [Doctrine SqlLogger](https://github.com/neos/flow-development-collection/blob/master/Neos.Flow/Classes/Persistence/Doctrine/Logging/SqlLogger.php) stopped working. 

The issue is that `$this->logger` is not an instance of `LoggerInterface` instead it is a `DependencyProxy`

```
/**
 * @var LoggerInterface
 */
protected $logger;

public function startQuery($sql, array $params = null, array $types = null)
{
    // this is a safeguard for when no logger might be available...
    if ($this->logger instanceof LoggerInterface) {
        ...
    }
}
```

I think this was caused by [BUGFIX: Use PSR-3 logger in SqlLogger](https://github.com/neos/flow-development-collection/commit/4015b65d1827f650faa48def7bd7fe559b113286) which means it only affects Flow 6.0.x

### Affected Versions
Flow: 6.0.x

**Comment by albe** [comment]
/fyi @daniellienert 

Was the (injected?) `LoggerInterface` always a `DependencyProxy` or is this newly so? If it always was, the fix is pretty easy.

**Comment by albe** [comment]
Should be fixed as of Flow 6.1.4 and 6.0.10
